{"status": "skipped", "description": " Checking if the position of objects changed during pick action i.e. Lucy collided with an obstacle.\n        ", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "pick_test"}, {"name": "subSuite", "value": "TestPickAction"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "1331-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.pick_test"}], "stop": 1607410303451, "uuid": "06c1727c-722a-49da-aea2-9e2f78447dcc", "historyId": "9ce2c514c4c415e484ddb826163036c0", "name": "test_allmodels_positon", "start": 1607410303451, "fullName": "tests.pick_test.TestPickAction#test_allmodels_positon", "statusDetails": {"message": "IOError: [Errno 2] No such file or directory: 'src/utilities/config.txt'", "trace": "self = <tests.pick_test.TestPickAction instance at 0x7f18ddb8aa28>\n\n    @pytest.fixture(autouse=True)\n    def set_up(self):\n>       self.config = Configuration()\n\ntests/pick_test.py:44: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.file_reader.file_reader.Configuration instance at 0x7f18ddb8ac20>\n\n    def __init__(self):\n    \n        self.rospakg       = 'atg'\n        self.ros_file_name = 'hsr.launch'\n    \n>       with open(\"src/utilities/config.txt\",mode='r') as file:\nE       IOError: [Errno 2] No such file or directory: 'src/utilities/config.txt'\n\ntests/file_reader/file_reader.py:32: IOError"}, "attachments": [{"source": "a0a9819f-abfd-4f94-b46a-17681fab6918-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "d49ffa3d-450a-40b1-b95c-731a1f743c9b-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "89dbb7f4-5af0-4bc7-aa16-02139177376d-attachment.txt", "type": "text/plain", "name": "stderr"}]}